web-api/terraform/template/elasticsearch.tf

```diff
@@ -15,4 +15,10 @@ resource "aws_elasticsearch_domain" "efcms-search" {
   snapshot_options {
     automated_snapshot_start_hour = 23
   }
+
+  log_publishing_options = {
```

Comment:
No need to use `=` here — I’m actually surprised this compiled, but blocks like this can be referenced by name without assigning them with equals (same as `snapshot_options`, `ebs_options`, and `cluster_config` above).